deep_field_metadetect/metacal.py

```diff
@@ -86,7 +88,15 @@ def get_max_gauss_reconv_psf(obs_w, obs_d, step=DEFAULT_STEP):
     return get_max_gauss_reconv_psf_galsim(psf_w, psf_d, step=step)
 
 
-def _render_psf_and_build_obs(image, obs, reconv_psf, weight_fac=1):
+def _render_psf_and_build_obs(
+    image, obs, reconv_psf, weight_fac=1, max_min_fft_size=None
```

Comment:
Let's call this `max_min_fft_size` just `fft_size` since it is used to set the FFT size directly. We should make this change everywhere.